dataset/dictionary/items.json5

```
@@ -1495,7 +1503,7 @@
     en: "Hydro Phantasm Taint-B-Gone",
     ja: "濁水幻霊イチコロリ",
     zhCN: "浊水幻灵克星",
-    pronunciationJa: "だくすいげんれいイチコロリ", // TODO Need Check
+    pronunciationJa: "だくすいげんれいイチコロリ",
```

Please revert this if there are no official sources.

```suggestion
    pronunciationJa: "だくすいげんれいイチコロリ", // TODO Need Check
```